src/app/ApplicationTemplate.UWP/Views/Content/Authentication/CreateAccountPage.xaml

```diff
@@ -263,8 +262,16 @@
 							Command="{Binding CreateAccount}"
 							Style="{StaticResource ExecutingButtonStyle}"
 							HorizontalAlignment="Stretch"
-							Margin="42,12,42,32" />
+							Margin="42,12,42,0" />
 			</StackPanel>
 		</ScrollViewer>
+
+		<!-- Back button -->
```

Comment:
Why are you using a custom back button? Why not use a CommandBar (which enables iOS back-swipe)?

Reply:
I used a custom button to follow the design. Is it preferred to use the command bar even if it doesn't follow exactly the design? 

Here is the current back button following the design  : 
![Capture3](https://user-images.githubusercontent.com/84537432/166847284-52b2ee2f-5852-4e09-b421-43d2d731eca6.PNG)

Here is the command bar native back button : 
![Capture](https://user-images.githubusercontent.com/84537432/166847277-39cdce0f-1e57-4e69-aab8-ce37d22fb361.PNG)

Reply:
For a back buttons, you should use the native one. That's what users expect.